feature-libs/product/multi-dimensional/components/multi-dimensional-container/product-multi-dimensional-container.component.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { isNotNullable, Product } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { distinctUntilChanged, filter, shareReplay } from 'rxjs/operators';
+import { CurrentProductService } from '@spartacus/storefront';
+
+@Component({
+  selector: 'cx-variants-multi-dimensional',
+  templateUrl: './product-multi-dimensional-container.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ProductMultiDimensionalContainerComponent {
```

Comment:
[MEDIUM]
1. `ProductMultiDimensionalContainerComponent` vs `ProductMultiDimensionalSelectorContainerComponent`?
(if yes, then let's align also filename)

1. Regardless of the above, for consistency let's align component `selector` with the class name

Reply:
See below comment.